pkg/storage/minio/checker.go

```diff
@@ -18,7 +18,7 @@ func (v *storageImpl) Exists(ctx context.Context, module, version string) (bool,
 	ctx, span := observ.StartSpan(ctx, op.String())
 	defer span.End()
 	versionedPath := v.versionLocation(module, version)
-	modPath := fmt.Sprintf("%s/go.mod", versionedPath)
+	modPath := fmt.Sprintf("%s.mod", versionedPath)
```

Comment:
iirc `versionedPath` is just the raw path, why is there no more need for the `go.mod` here?